tests/plugins/tableselection/manual/draganddropimage.md

```diff
@@ -0,0 +1,17 @@
+@bender-tags: dialog, __next__, 547, bug
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, table, tableselection, image, undo, sourcearea, basicstyles
```

Comment:
Please add `elementspath` plugin as it additionally shows where is the current selection. The same for the other manual test.